raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -47,7 +48,7 @@ contract NettingChannelContract {
         (success, balance) = data.deposit(amount);
 
         if (success == true) {
-            ChannelNewBalance(data.token, msg.sender, balance, data.opened);
+            ChannelNewBalance(data.token, msg.sender, balance, block.number);
```

Comment:
yeah, but until then this change will at least prevent errors from using the block number we provide. I could change it to some illegal value if you'd prefer that?